draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -6921,32 +6921,45 @@ Vary: accept-encoding, accept-language
    the "lost update" problem: one client accidentally overwriting
    the work of another client that has been acting in parallel.
 </t>
-<t><iref primary="false" item="selected representation"/>
-   Conditional request preconditions are based on the state of the target
-   resource as a whole (its current value set) or the state as observed
-   in a previously obtained representation (one value in that set).
-   A resource might have multiple current representations, each with its
-   own observable state.  The conditional request mechanisms assume that
-   the mapping of requests to a <x:ref>selected representation</x:ref> (<xref target="representations"/>)
-   will be consistent over time if the server intends to take advantage of
-   conditionals. Regardless, if the mapping is inconsistent and the server is
-   unable to select the appropriate representation, then no harm will result
-   when the precondition evaluates to false.
-</t>
 
 <section title="Preconditions" anchor="preconditions">
+   <iref primary="false" item="selected representation"/>
+<t>
+   Preconditions are usually defined with respect to a state of the target
+   resource as a whole (its current value set) or the state as observed in a
+   previously obtained representation (one value in that set). If a resource
+   has multiple current representations, each with its own observable state,
+   a precondition will assume that the mapping of requests to a
+   <x:ref>selected representation</x:ref> (<xref target="representations"/>)
```

Comment:
The differing pluralisation in 'mapping of requests to a selected representation' tripped me up a bit. Suggest: 

```suggestion
   a precondition will assume that the mapping of requests to their
   <x:ref>selected representations</x:ref> (<xref target="representations"/>)
```

Reply:
```suggestion
   a precondition will assume that the mapping of each request to a
   <x:ref>selected representation</x:ref> (<xref target="representations"/>)
```